lib/matplotlib/axes/_axes.py

```diff
@@ -353,7 +329,7 @@ def legend(self, *args, **kwargs):
             ``bbox_to_anchor`` will be ignored).
 
         bbox_to_anchor : :class:`matplotlib.transforms.BboxBase` instance \
-or tuple of floats
+        or tuple of floats
```

Comment:
This should not be indented (otherwise the rendering in ipython gets messed up).